server/v1/controllers/auth.controller.js

```diff
@@ -41,6 +43,42 @@ class userController {
       });
     }
   }
+
+  static signIn(req, res) {
+    const { error } = validate.validateSignin(userLogin(req));
+
+    if (error) {
+      return res.status(400).json({
+        status: 400,
+        errorMessage: error.details[0].message.replace(/"/g, '')
+      });
+    }
+
+    const userAccount = users.find(usr => usr.email === req.body.email);
+
+    if (!userAccount) {
+      return res.status(401).json({
+        status: 401,
+        message: "Oops, You don't have an account yet, Please sign up"
+      });
+    }
+    const verifyPassword = auth.checkPassword(
+      req.body.password,
+      userAccount.password
+    );
+    if (verifyPassword) {
+      return res.status(202).json({
+        status: 202,
```

Comment:
Please check this link to know which status code to use.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status